### PR TITLE
Dashboard delete

### PR DESCRIPTION
**Pull request by vrackovic**
- Added functionality for deleting tests from dashboard component

### CODE REVIEW

File: web/src/main/frontend/src/components/ui/TableBody.js [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
+class TableBody extends React.Component {
+  rowDelete = row => () => {
+    this.props.dashboard.deleteTest(row.id);
+  };
+
+  render() {
+    return this.props.tableRows.map(row => (
+      <tr
+        key={row.id}
+        onClick={this.props.rowClick ? () => this.props.rowClick(row) : null}
```
Comment (iscekic) [inline web/src/main/frontend/src/components/ui/TableBody.js]: Check when `rowClick` is false.

Comment (iscekic) [reply web/src/main/frontend/src/components/ui/TableBody.js]: Also no need to wrap `rowClick` in anonymous arrow fn.

Comment (vrackovic) [reply web/src/main/frontend/src/components/ui/TableBody.js]: `rowClick` doesn't exist in `Dashboard` component, so it is passed as `undefined` when triggered from there.

**Comment by vrackovic** [comment]
Closing pull request until login is properly implemented

File: web/src/main/frontend/src/components/ui/TableBody.js [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
+      >
+        {this.props.tableHeadings.map(heading => (
+          <React.Fragment key={heading}>
+            <td>
+              {heading !== 'email' ? capitalize(row[heading]) : row[heading]}
+            </td>
+          </React.Fragment>
+        ))}
+        {this.props.isAdmin && (
```
Comment (iscekic) [inline web/src/main/frontend/src/components/ui/TableBody.js]: Maybe this can be done in a better way... Create an improvement task please.

Comment (vrackovic) [reply web/src/main/frontend/src/components/ui/TableBody.js]: Sure :smile:

**Pull request merged**